packages/utils/crates/utils-types/src/primitives.rs

```diff
@@ -377,7 +377,7 @@ impl Balance {
     pub const SIZE: usize = U256::SIZE;
 
     /// Number of digits in the base unit
-    pub const SCALE: usize = 18;
+    pub const SCALE: usize = 19;
```

Comment:
I get that this fixes an issue on balance format, but it's quite confusing to see that the "Number of digits in the base unit" differs from the "Token decimal" by 1. Shouldn't it be `18` here, and `SCALE+1` when used ?

Reply:
Possibly, feel free to provide an improvement on `master`.